xline/src/main.rs

```diff
@@ -183,11 +184,11 @@ struct ServerArgs {
     #[clap(long, value_parser = parse_duration, default_value = "50ms")]
     rpc_timeout: Duration,
     /// Candidate election timeout
-    #[clap(long, value_parser = parse_duration, default_value = "1s")]
-    candidate_timeout: Duration,
+    #[clap(long, default_value_t = default_candidate_timeout_ticks())]
+    candidate_timeout: u8,
```

Comment:
rename this as `candidate_timeout_ticks`

Reply:
Fixed.